src/clj/game/cards-ice.clj

```diff
@@ -336,9 +337,10 @@
                                       (->> (vec (.split (:subtype card) " - "))
                                            (concat [target])
                                            (join " - "))))
+                      (update! (assoc card :subtype-target target))
```

Comment:
So `(get-card state card)` refreshes the card as defined in the state (so that could have been an alternative solution for the problem above). It's been added in a lot of places recently to prevent client side hacking of trash / rez / play costs or other devious things by reseting the card definition to what the server has. Except for client-facing functions there is normally not much need to call `(get-card state card)` since its often clearer to use a `let` or similar to define the "updated" card before you send it to `update!` or other similar function.

@nealterrell anything to add to this?